torchx/mix.exs

```diff
@@ -43,4 +49,113 @@ defmodule Torchx.MixProject do
       source_url: @source_url
     ]
   end
+
+  defp compilers do
+    if System.get_env("LIBTORCH_DIR"), do: [:elixir_make], else: [:torchx, :elixir_make]
+  end
+
+  defp aliases do
+    [
+      "compile.torchx": &compile/1
+    ]
+  end
+
+  defp compile(args) do
+    cache_dir = Path.dirname(@libtorch_cache)
+
+    if "--force" in args do
+      File.rm_rf!(cache_dir)
+    end
+
+    if File.dir?(@libtorch_cache) do
+      {:ok, []}
+    else
+      install_libtorch(cache_dir)
+    end
+  end
+
+  defp install_libtorch(cache_dir) do
+    File.mkdir_p!(cache_dir)
+
+    libtorch_zip =
+      System.get_env(
+        "LIBTORCH_ZIP",
+        Path.join(cache_dir, "libtorch-#{@libtorch_version}-#{@libtorch_target}.zip")
+      )
```

Comment:
```suggestion
  defp compile(args) do
    libtorch_cache = @libtorch_cache
    cache_dir = Path.dirname(libtorch_cache)

    if "--force" in args do
      File.rm_rf!(cache_dir)
    end

    if File.dir?(libtorch_cache) do
      {:ok, []}
    else
      install_libtorch(cache_dir, libtorch_cache)
    end
  end

  defp install_libtorch(cache_dir, libtorch_cache) do
    File.mkdir_p!(cache_dir)
    libtorch_zip = libtorch_cache <> ".zip"
```

Reply:
In particular, note that setting LIBTORCH_ZIP doesn't quite work if you set it to anything outside of the cache dir, so we should go ahead and remove it for now. :)